PhysicsTools/NanoAOD/python/nano_eras_cff.py

```diff
@@ -27,3 +27,5 @@
 from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
 
 from Configuration.Eras.Modifier_tracker_apv_vfp30_2016_cff import tracker_apv_vfp30_2016
+
+from Configuration.Eras.Modifier_run2_JMENanoHerwig7_cff import Modifier_run2_JMENanoHerwig7
```

Comment:
```suggestion
from Configuration.Eras.Modifier_run2_JMENanoHerwig7_cff import run2_JMENanoHerwig7
```
This suggests me that this PR was never actually tested: could you please apply the fix and then make a test run with the modifier enabled, and verify that it finally works as expected?